Description: Fix hyperlink to tutorial_reference.pdf.
 Fix a hyperlink in the documentation, to cope with the fact that
 tutorial_reference.pdf is compressed in the binary package.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-03-02

--- a/docsrc/index.dxx
+++ b/docsrc/index.dxx
@@ -560,7 +560,7 @@
     
     In the same way, you get documentation for the individual functions. More 
     information about the glue code behind the bindings is available in 
-    <a href="documents/tutorial_reference.pdf">this report</a>.
+    <a href="documents/tutorial_reference.pdf.gz">this report</a>.
     
 */
 
